src/vellum/workflows/nodes/displayable/web_search_node/tests/test_node.py

```diff
@@ -0,0 +1,323 @@
+import pytest
+from unittest.mock import MagicMock
+
+import requests
+
+from vellum.workflows.errors.types import WorkflowErrorCode
+from vellum.workflows.exceptions import NodeException
+from vellum.workflows.inputs import BaseInputs
+from vellum.workflows.state import BaseState
+from vellum.workflows.state.base import StateMeta
+
+from ..node import WebSearchNode
+
+
+@pytest.fixture
+def mock_context():
+    """Create a mock context with vellum_client."""
+    context = MagicMock()
+    context.vellum_client._client_wrapper.get_headers.return_value = {"User-Agent": "vellum-python-sdk/1.0.0"}
```

Comment:
[nit] we have a `vellum_client` fixture that we're using that we could add this support to and use instead